apis/swagger.yml

```diff
@@ -337,6 +337,35 @@ paths:
           schema:
            $ref: "#/definitions/Error"
       tags: ["Container"]
+  /containers/{id}/start:
+    post:
+      summary: "Start a container"
+      operationId: "ContainerStart"
+      parameters:
+        - name: "id"
+          in: "path"
+          required: true
+          description: "ID or name of the container"
+          type: "string"
+        - name: "detachKeys"
+          in: "query"
+          description: "Override the key sequence for detaching a container. Format is a single character `[a-Z]` or `ctrl-<value>` where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`."
+          type: "string"
+      responses:
+        204:
```

Comment:
If this this 204, then I think we need to update https://github.com/alibaba/pouch/blob/master/apis/server/container_bridge.go#L129 to be `resp.WriteHeader(http.StatusNoContent)`.